profiles/kentik_snmp/hpe/hpe-synergy.yml

```diff
@@ -0,0 +1,60 @@
+#https://mibs.observium.org/mib/SYNERGY100G-HPE-CPU-UTIL-MIB/
+#https://mibs.observium.org/mib/SYNERGY100G-HPE-NLP-MIB/#syn100GhpeNLPModuleConfig
+---
+extends:
+  - system-mib.yml
+  - if-mib.yml
+  - ip-mib.yml
+
+sysobjectid:
+  - 1.3.6.1.2.1.11.5.7.5.9    # VC SE 100Gb F32 Module
+
+
+provider: kentik-server-chassis
+
+metrics:
+  - MIB: HPE-SYNERGY-MIB
+    symbols:
+      - name:  syn100GhpeSwitchMaxCPUThreshold
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4130.1.1
+        tag: MAX_CPU
+        poll_time_sec: 60
+      - name:  syn100GhpeSwitchMinCPUThreshold
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4130.1.2
+        tag: MIN_CPU
+        poll_time_sec: 60
+      - name:  syn100GhpeSwitchAverageCPUUtilization  
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4130.2.1
+        tag: AVG_CPU
+        poll_time_sec: 60
```

Comment:
2 issues in this section

1. there is an extra space between the key and value on all of the `- name:` attributes here
2. these tags will result in metrics like this:

```
kentik.snmp.MAX_CPU
kentik.snmp.MIN_CPU
kentik.snmp.AVG_CPU
```

If this is expected, then please change them to lower-case. 

If the intention is to simply report `kentik.snmp.CPU` like all other profiles do, then just use the `CPU` tag on the `syn100GhpeSwitchAverageCPUUtilization` OID.